Formula/hypre.rb

```diff
@@ -14,20 +15,10 @@ class Hypre < Formula
 
   depends_on "gcc" # for gfortran
   depends_on "open-mpi"
-  depends_on "veclibfort"
 
   def install
     cd "src" do
-      ENV["CC"] = ENV["MPICC"]
-      ENV["CXX"] = ENV["MPICXX"]
-
       system "./configure", "--prefix=#{prefix}",
-                            "--with-blas=yes",
-                            "--with-blas-libs=blas cblas",
-                            "--with-blas-lib-dirs=/usr/lib",
-                            "--with-lapack=yes",
-                            "--with-lapack-libs=lapack clapack f77lapack",
-                            "--with-lapack-lib-dirs=/usr/lib",
```

Comment:
hypre ships its own sources for blas and lapack. It can be told to use system libs, but in my experimentation it was challenging to convince it to link against openblas, and it is shipping static libraries by default anyway, so I'm really not sure what the point of linking in OpenBLAS or veclibfort/Accelerate is. You can turn on dynamic libs, but then it doesn't ship a static lib.